includes/admin/settings/display-settings.php

```diff
@@ -22,7 +22,13 @@
  */
 function edd_options_page() {
 	$active_tab = isset( $_GET['tab'] ) && array_key_exists( $_GET['tab'], edd_get_settings_tabs() ) ? $_GET['tab'] : 'general';
-
+	$sections = edd_get_settings_sections( $active_tab );
+	$key = 'main';
+	if ( is_array( $sections ) ) {
+		$key = key( $sections );
+	}
+	
+	$section = isset( $_GET['section'] ) && array_key_exists( $_GET['section'], edd_get_settings_sections( $active_tab ) ) ? $_GET['section'] : $key;	
```

Comment:
Looks like I am returning false instead of an empty array... I will try to push out a fix tonight, either here or in register-settings.php